openapi_test.go

```diff
@@ -201,6 +216,17 @@ func Test_tagFromType(t *testing.T) {
 			}
 		})
 	}
+
+	t.Run("struct with nested tags", func(t *testing.T) {
+		s := NewServer()
+		tag := SchemaTagFromType(s.OpenAPI, MyStructWithNested{})
+		nestedProperty := tag.Value.Properties["nested"]
+		require.Equal(t, "my struct", nestedProperty.Value.Description)
+		require.Equal(t, "my description", nestedProperty.Value.Properties["c"].Value.Description)
+		require.Equal(t, 8, nestedProperty.Value.Properties["c"].Value.Example)
+		require.Equal(t, float64(3), *nestedProperty.Value.Properties["c"].Value.Min)
+		require.Equal(t, float64(10), *nestedProperty.Value.Properties["c"].Value.Max)
```

Comment:
This would panic if c is not present


```suggestion
		require.Contains(t, nestedProperty.Value.Properties, "c")
		require.Equal(t, "my description", nestedProperty.Value.Properties["c"].Value.Description)
		require.Equal(t, 8, nestedProperty.Value.Properties["c"].Value.Example)
		require.Equal(t, float64(3), *nestedProperty.Value.Properties["c"].Value.Min)
		require.Equal(t, float64(10), *nestedProperty.Value.Properties["c"].Value.Max)
```

Reply:
Updated